File: src/main/java/com/learnershi/rclasssocket/entity/RandomPick.java
package com.learnershi.rclasssocket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

/**
 * Random
 * @author Sunhayng
 */
@Accessors(chain = true)
@NoArgsConstructor
@Getter
@Setter
@Document(collection = "randomPick")
public class RandomPick extends ClassRoomEntity {
    private List randomPickList;  
    private Boolean isShuffle;
    private String pickName;
}

File: src/main/java/com/learnershi/rclasssocket/entity/RandomPick.kt
package com.learnershi.rclasssocket.entity

import org.springframework.data.mongodb.core.mapping.Document

/**
 * Random
 *
 */
@Document(collection = "randomPick")
class RandomPick : ClassRoomEntity() {
    private val randomPickList: List<*>? = null
    private val isShuffle: Boolean? = null
    private val pickName: String? = null
}